server/__test__/db_test/index.test.js

```diff
@@ -0,0 +1,13 @@
+const connection = require("../../database/config/connection");
+const build = require("../../database/config/build");
+const { insertFoodQuery } = require("../../database/queries");
+
+beforeEach(() => build());
+afterAll(() => connection.end());
+
+describe("Test insertFood Query", () => {
+  test(" Test insert Food ", async() => {
+    const {rowCount} = await insertFoodQuery(1, 1, 1, 3)
```

Comment:
Hello Khamis, I think it's good to make more readable code as some variables here to explain what you wrote in this query params, whats these numbers mean !!

Reply:
Ok, but here I'm making a database test so I should replace the values

Reply:
Oh,sorry khamis, Iam not writing my comment correctly , I mean just to add make like this :-
```js
const userId=1;
const foodId = 1;
const categoryId=1;
const grams=3'

 const {rowCount} = await insertFoodQuery(userId, foodId, categoryId, grams)
```
I think by doing this will be more readable :1st_place_medal: